examples/otlp/README.md

```diff
@@ -7,6 +7,7 @@ Protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/mai
 The application in `grpc_main.cc` initializes an `OtlpGrpcExporter` instance,
 the application in `http_main.cc` initializes an `OtlpHttpExporter` instance.
 The application in `http_log_main.cc` initializes an `OtlpHttpLogExporter` instance
+The application in `grpc_log_main.cc` initializes an `OtlpGrpcLogExporter` instance
```

Comment:
nit: fix period in line 9 and 10, or fix the whole sentence from line 7 to 10.

Reply:
done